### DISCUSSION TITLE
(Lesson 2)  Error reported while executing "from tvm.script import tir as T"

### DISCUSSION THREAD

**Discussion by tfruan2000** [mlc-ai/mlc-zh#109] [Q&A] [answered]
Error reported while executing "from tvm.script import tir as T" 
**AttributeError: module 'tvm.arith._ffi_api' has no attribute 'CreateAnalyzer'** 

my environment: python3.10.5, win 11

**Comment by Hzfengsy** [comment, marked as answer]
Windows wheel可能有点问题，正在排查。但目前课程组成员均没有windows环境，可能仍然需要一定的时间。在此之前可以考虑：
1. 使用Linux环境或WSL环境
2. 手动clone 代码库https://github.com/mlc-ai/relax 本地编译
3. 使用colab在线环境

对带来的不便感到抱歉

**Reply by tfruan2000** [reply]
使用colab能暂时解决问题，谢谢！
